.github/workflows/api-tests.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will Run API tests using Newman
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ develop ]
```

Comment:
@wgu-edwin For testing purposes can u please change the name of the branch to "fix/GH-runner-api-tests" instead of "develop"
That makes sure the builds are executing when we do any checkins to this feature branch.

Reply:
Does the section below for `pull_request` address your request? Any PRs against `develop` should run this workflow, and this PR is against `develop`

If you look on the 'Actions' tab above in the PR there is a second 'Java CI with Maven' that if you expand its dropdown shows `api-test` is executing for this PR.

Please clarify if whether this is what you are asking for or if Edwin should also change the `push` too for assisting with review.

Reply:
'Java CI with Maven' in Actions demonstrates that this github workflow is running fine.
Would have been helpful if it was added in the PR description.
If Edwin is confident it works without any issues, PR can be approved.

Reply:
@schagantiwgu if you think that PR descriptions and commit messages are helpful, please help make this a quality improvement for the team? Make it a retro item?